### PR TITLE
add cups for teams.

### PR DESCRIPTION
**Pull request by troy2914**

### CODE REVIEW

File: cups/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+# Define here handlers associated to this role.
+
```
Comment (vmcj) [inline cups/handlers/main.yml:3]:
```suggestion
```

File: cups/templates/printers.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Printer configuration file for CUPS v1.7.2
+# Written by cupsd
+# DO NOT EDIT THIS FILE WHEN CUPSD IS RUNNING
+<DefaultPrinter contest>
+UUID urn:uuid:db002b1c-c421-3c4b-46da-d0e7d4d2ebb4
+AuthInfoRequired none
+Info contest
+DeviceURI ipp://printsvr:631/classes/contest
```
Comment (vmcj) [inline cups/templates/printers.conf.j2:8]: Maybe a typo?
```suggestion
DeviceURI ipp://printsrv:631/classes/contest
```

File: lastminutewf2020.yml [hunk 3]
```diff
@@ -57,6 +56,13 @@
         owner: root
         group: root
         mode: '0644'
+    - name: Ensure printsrv in /etc/hosts
+      ansible.builtin.lineinfile:
+        path: /etc/hosts
+        line: "{{ baseip }}.211	printserv"
```
Comment (vmcj) [inline lastminutewf2020.yml:62]:
```suggestion
        line: "{{ baseip }}.211	printserver"
```

File: lastminutewf2020.yml [hunk 4]
```diff
@@ -57,6 +56,13 @@
         owner: root
         group: root
         mode: '0644'
+    - name: Ensure printsrv in /etc/hosts
```
Comment (vmcj) [inline lastminutewf2020.yml:59]:
```suggestion
    - name: Ensure printserver in /etc/hosts
```

File: cups/templates/printers.conf.j2 [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+PPDTimeStamp *
+State Idle
+StateTime 1463533936
+Type 8388614
+Accepting Yes
+Shared No
+ColorManaged Yes
+JobSheets {{ job_sheet}} none
```
Comment (vmcj) [inline cups/templates/printers.conf.j2:16]:
```suggestion
JobSheets {{ job_sheet }} none
```
The convention is to have spaces there,

You could try to run ansible-lint on the role for these conventions, most of the time I can agree with these conventions.

**Review by vmcj — APPROVED** [review]
Added some suggestions, but I think this should work.

File: cups/templates/printers.conf.j2 (hunk 2 above)
Comment (troy2914) [reply cups/templates/printers.conf.j2:8]: no printsrv is the correct,  lastminute2022.yml was creating the wrong name.

File: lastminutewf2020.yml (hunk 3 above)
Comment (troy2914) [reply lastminutewf2020.yml:62]: actually it should be printsrv,  i have removed that extra e

File: lastminutewf2020.yml (hunk 4 above)
Comment (troy2914) [reply lastminutewf2020.yml:59]: nope, hostname should be printsrv.

**Pull request merged**